Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -30,10 +38,50 @@ public Task<bool> Execute(EditUserRequest request)
             {
                 passwordChangeValidator.ValidateAndThrow(passwordRequest);
             }
+
             userInfoValidator.ValidateAndThrow(userInfo);
```

Comment:
Let's catch this and throw BadRequest exception, I think that client should receive only custom exceptions

Reply:
Fixed